src/phase_utils.py

```diff
@@ -205,8 +208,80 @@ def __str__(self):
         return f"lp-{self._id}"
 
 
+class AnnotationStation:
+    """
+    Used for saving label annotations for songs.
+    Reducing the need for annotating the same song more than once.
+
+    Annotations will be saved as a dictionsary in following format
+    for song ids 1 and 2::
+
+        {'1': [[arousal1], [valence1]], '2': [[arousal2], [valence2]]}
+    """
+
+    def __init__(self, path: Path):
+        """
+        AnnotationStation constructor.
+
+        Args:
+            path (Path): Path to dictionary in json format.
+                    (Note: include `.json` tag.)
+
+        Raises:
+            FileNotFoundError: When path is not correct.
+        """
+        self.path = path
```

Comment:
`self.path` should use '_' to indicate an internal class variable:
`self._path`. 
Same goes for all class variables.